src/common/utils.ts

```diff
@@ -105,20 +106,32 @@ function xrpToDrops(xrp: BigNumber.Value): string {
     .toString(10)
 }
 
-function toRippledAmount(amount: Amount): RippledAmount {
+function toRippledAmount(amount: RippledAmount): RippledAmount {
+  if (typeof amount === 'string')
+    return amount;
+
   if (amount.currency === 'XRP') {
     return xrpToDrops(amount.value)
   }
   if (amount.currency === 'drops') {
     return amount.value
   }
+
+  let issuer = amount.counterparty || amount.issuer
+
+  let tag
```

Comment:
`let` is always a code smell to me. you should be able to get rid of this by making your `try/catch` its own function

Reply:
I disagree here. IMO this is a reasonable use of `let`.

It allows it to be assigned to once in a different scope, and then is immediately tested afterwards in the same scope it was declared. 

If you feel strongly, lets chat about it in person 😄

Reply:
Though, because I went back to look at this I caught a bug w/ `tag: 0`.

Reply:
no - i don't feel _strongly_, but whenever I find myself writing a `let` I think about whether it can be refactored. I agree, your usage here is reasonable but it also made me question it to determine that it was reasonable, if that makes sense.